docker/local/docker-compose-dev.yml

```diff
@@ -0,0 +1,87 @@
+version: "3.2"
```

Comment:
It is a copy of an existing docker compose file as I did not want to mess with the existing setup.

It has been used for development using the 'docker compose watch' feature (see lines 68 - 72, more details on watch here: https://docs.docker.com/compose/how-tos/file-watch/).

docker compose watch significantly speeds up the inner dev loop as it constantly syncs any code changes to the running container.

Not required that we keep it. Can delete it in case you feel so.
However, in case you reckon it is worthwhile keeping it around then we could add some documentation similar to what we added to the GRR docs (see [Section 4 here](https://docs.docker.com/compose/how-tos/file-watch/) for an example).

Reply:
@hacktobeer Do you have any thoughts on this one?  Do you think it would be beneficial to keep this one, or possibly integrate the relevant changes to one of the other existing dev compose files?